src/modules/appData/appData-module.md

```diff
@@ -23,3 +23,4 @@ This module will expose:
 
 - `useAppData()`: Exposes the app hash and the document associated to it.
 - `AppDataUpdater`: Component that makes sure the appData has the right content, required for `useAppData` hook to work
+- `useUploadAppData()`: Returns a function which can be invoked when an `appData` should be uploaded. This will be uploaded to IPFS. In case of errors, it will keep re-attempting.
```

Comment:
As it'll be send to backend instead and potentially later uploaded to IPFS, I suggest removing the mention to where it's uploaded to.

```suggestion
- `useUploadAppData()`: Returns a function which can be invoked when an `appData` should be uploaded. In case of errors, it will keep re-attempting.
```